DiscordChatExporter.Core/Exporting/ExportRequest.cs

```diff
@@ -148,12 +149,12 @@ public partial class ExportRequest
             {
                 "%g" => guild.Id.ToString(),
                 "%G" => guild.Name,
-                "%t" => channel.Category.Id.ToString(),
-                "%T" => channel.Category.Name,
+                "%t" => channel.ParentId.ToString(),
+                "%T" => channel.ParentName,
```

Comment:
```suggestion
                "%T" => channel.ParentName ?? "",
```

Does this fix things?

Reply:
Seems to have fixed it, yeah! Thanks a ton!

Reply:
Resolved in 9ed1238.